src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -85,8 +85,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
         if (false !== strpos($input->getFirstArgument(), ':d')) {
-            $output->writeln('<comment>The use of "translation:debug" command is deprecated since version 2.7 and will be removed in 3.0. Use the "debug:translation" instead.</comment>');
+            $output->newLine();
```

Comment:
Actually, all those `newLine` calls are here in order to have the command output preceded and followed by 1 blank line, to increase readability and separate output of different commands, which is not fully handled by SymfonyStyle.  
Do you still want me to remove them ?